docs/topic_install-f5-agent.rst

```diff
@@ -30,8 +30,8 @@ The ``f5-openstack-agent`` package can be installed using ``dpkg`` tools.
 .. code-block:: bash
 
     $ curl -L -O https://github.com/F5Networks/f5-common-python/releases/download/v1.5.0/python-f5-sdk_1.5.0-1_1404_all.deb
-    $ curl -L -O https://github.com/F5Networks/f5-icontrol-rest-python/releases/download/v1.1.0/python-f5-icontrol-rest_1.1.0-1_1404_all.deb
-    $ sudo dpkg –i python-f5-icontrol-rest_1.1.0-1_1404_all.deb
+    $ curl -L -O https://github.com/F5Networks/f5-icontrol-rest-python/releases/download/v1.0.9/python-f5-icontrol-rest_1.0.9-1_1404_all.deb
```

Comment:
We'll have to keep the newer version of icontrol rest, which is 1.1.0, since SDK 2.0.0 relies upon it and it has the policy logic.